### ISSUE TITLE
Polish language does not have proper localized string for "no search results" text

### ISSUE THREAD

**Issue by ek68794998** [Azure/LogicAppsUX#3972]
### Describe the Bug with repro steps 

1. Set language to Polish 
2. Open connector search panel 
3. Type anything 

The panel will error out.

### What type of Logic App Is this happening in? 

Power Automate

### Are you using new designer or old designer 

New Designer

### Did you refer to the TSG before filing this issue? [https://aka.ms/lauxtsg](<https://aka.ms/lauxtsg>) 

Yes

### Workflow JSON 

*No response*

### Screenshots or Videos 

![image](<https://github.com/Azure/LogicAppsUX/assets/1350074/ee5f522a-1117-4259-956f-25d2c5c40a2c>)

### Browser 

Chrome 120, Windows 11.

### Additional context 

The Polish language string is not up to date; it is still using `{searchText}` interpolation instead of `{searchTerm}`.

**Comment by ek68794998** [comment]
For easy reference, this string is from `libs\designer-ui\src\lib\panel\recommendationpanel\searchResult.tsx:83`.

**Comment by ek68794998** [comment]
We're also seeing an issue with Dutch in a different string. It looks like Dutch is using `{methode}` rather than the correct `{method}`:

![image](https://github.com/Azure/LogicAppsUX/assets/1350074/d678150d-cd20-4b55-8031-88f1bba33b47)

Here's Russian, for comparison, which works fine:

![image](https://github.com/Azure/LogicAppsUX/assets/1350074/263aec3c-28f9-4888-a0db-feb600b6998e)

**Comment by hartra344** [comment]
Those aren’t supposed to be translated >.< they’re variables that get replaced. @ek68794998 if you see any more of these feel free to fix them in the strings.x.json file as well as the lcl file. The fix is to just change them to the original English string.

**Comment by hartra344** [comment]
Also in the future, please just link the file where it's happening instead of an image of the file. It makes it easier to search files.

**Comment by ek68794998** [comment]
@hartra344 Thanks! Will do in the future.

Looks like you fixed the Polish issue in 478b56636047cc1267a8cb3070dc5e9572fb9642 but the Dutch one is still outstanding so I will make a PR for that.

**Comment by ek68794998** [comment]
@hartra344 It looks like the change in 478b56636047cc1267a8cb3070dc5e9572fb9642 has been undone.

In cb1a03abc7efa3a09b31c85ab6fa4b637700e2e3, it got converted back to `{searchText}`, so we're seeing the error come up again.

![image](https://github.com/Azure/LogicAppsUX/assets/1350074/e5c7c980-9bfb-408f-81ef-10a880b4313e)

It also looks like the fix I made in #3978 has been undone. I didn't track back in history to find when it happened, but the current `strings.nl.json` has been changed back to `{methode}`:

![image](https://github.com/Azure/LogicAppsUX/assets/1350074/c2c1f96e-7e57-4c8e-8ff2-c3cb0a3b077a)

Any way we can get these changes to be sticky?
